uberduck_ml_dev/data_loader.py

```diff
@@ -159,6 +176,13 @@ def __len__(self):
             return min(self.debug_dataset_size, len(self.audiopaths_and_text))
         return len(self.audiopaths_and_text)
 
+    def sample_test_batch(self, size):
+        idx = np.random.choice(range(len(self)), size=size, replace=False)
+        test_batch = []
+        for index in idx:
+            test_batch.append(self.__getitem__(index))
```

Comment:
minor question, is `self[index]` more idiomatic?

Reply:
It is but this function came directly from the GradTTS repo: https://github.com/huawei-noah/Speech-Backbones/blob/main/Grad-TTS/data.py#L71-L76